app/graphql/mutations/sign_up.rb

```diff
@@ -6,9 +6,13 @@ class SignUp < BaseMutation
     argument :first_name, String, required: false
     argument :last_name, String, required: false
 
+    argument :avatar, Types::ImageUploaderType, required: false
```

Comment:
```suggestion
    argument :avatar, Types::ImageUploaderType, as: :avatar_data, required: false
```

We can use alias if shine accepts avatar_data as parameters.

Reply:
Also, did you investigated the ability to write a spec for this?

Reply:
Can you check this
```ruby
argument :avatar, Types::ImageUploaderType, required: false, prepare: :to_h
```

Reply:
I added avatar data to test for SignUp mutation